### PR TITLE
TW-295 Add Find and replace to /getting-started/installation-and-updates/#web-limitations 

### PR DESCRIPTION
**Pull request by HeatherStenson**
- Adds "Find and replace" to the list of limitations for the Postman web client
- Removes "Live preview" and "Saving responses to file" from the list of limitations for the Postman web client (both options are available on web now)
- General copyedits

### CODE REVIEW

File: src/pages/docs/getting-started/installation-and-updates.md [hunk 1]
```diff
@@ -126,7 +127,7 @@ You can use Postman in your web browser to carry out your API development and te
 
 ### Browser requirements
 
-Postman’s web application is optimized for the following browsers:
+Postman’s web client is optimized for the following browsers:
```
Comment (jkonrath-postman) [inline src/pages/docs/getting-started/installation-and-updates.md:130]: Take a look at TW-188 - can we change these to Postman for Web?

File: src/pages/docs/getting-started/installation-and-updates.md [hunk 2]
```diff
@@ -135,30 +136,32 @@ Postman’s web application is optimized for the following browsers:
 
 ### Installing the Postman desktop agent
```
Comment (jkonrath-postman) [inline src/pages/docs/getting-started/installation-and-updates.md:137]: Let's talk about the naming of this offline...

**Review by jkonrath-postman — CHANGES_REQUESTED** [review]
Some minor naming things, the agent one in particular. I think that needs some more research.

File: src/pages/docs/getting-started/installation-and-updates.md [hunk 3]
```diff
@@ -64,8 +65,8 @@ Postman is available for macOS 10.11 (El Capitan) and later.
 
     > Make sure to download the **Mac Apple Chip** version if you have a Mac with the M1 processor.
 
-1. If your browser downloads the file as a ZIP file, double-click the file in the **Downloads** folder to unzip it.
-1. In the **Downloads** folder, double-click the `Postman` file to install it.
+1. If your browser downloads the file as a ZIP file, find the file in the **Downloads** folder and unzip it.
+1. In the **Downloads** folder, install the unzipped `Postman` file.
```
[the remark this answers is not in the document]
Comment (HeatherStenson) [reply src/pages/docs/getting-started/installation-and-updates.md]: Yeah, this ended up funkier than I intended. I think it's best to return to the original wording here - changed in 3d8f2c1

File: src/pages/docs/getting-started/installation-and-updates.md (hunk 1 above)
Comment (HeatherStenson) [reply src/pages/docs/getting-started/installation-and-updates.md]: Updated in c423aa3

**Review by mark-anthony-postman — APPROVED** [review]
LGTM 🙂

**Review by jkonrath-postman — APPROVED** [review]
LGTM, ignore my comments on naming

**Pull request merged**